src/board/audio/index.ts

```diff
@@ -138,13 +138,7 @@ export class Audio {
     }
   }
 
-  boardStopped() {
-    if (this.context) {
-      this.context.close();
-      this.context = undefined;
-      this.oscillator = undefined;
-    }
```

Comment:
Previously thrown away when the board was stopped, but not re-initialised on a soft reboot. This is now thrown away when `stop()` is called.

Reply:
Can this mean we keep playing sounds? Would it be better to close/reinit the audio?

Reply:
This doesn't appear to be the case. It may be safer, but we need to find a way to pass  `wrapped._microbit_hal_audio_ready_callback` and `wrapped._microbit_hal_audio_speech_ready_callback` into an init function that is called on a soft reboot.